habitat-aspnet-sample/hooks/init

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+#
+ln -sf {{pkg.path}}/www {{pkg.svc_var_path}}
```

Comment:
oh right! I just started with the tutorial app plan and was gonna change this all up after linting (clearly I forgot) and I suppose we should edit the tutorial app to comply.